.gitattributes

```diff
@@ -11,5 +11,8 @@ sdk/python/pyproject.toml linguist-generated=false
 
 sdk/typescript/**/* linguist-generated=true
 sdk/typescript/esc/index.ts linguist-generated=false
+sdk/typescript/esc/workspace.ts linguist-generated=false
+sdk/typescript/esc/workspace_models.ts linguist-generated=false
 sdk/typescript/test/**/* linguist-generated=false
 sdk/typescript/* linguist-generated=false
+sdk/test/**/* linguist-generated=false
```

Comment:
this line isn't necessary since everything is false by default. the only reason we specify the others above as false is because of the wildcard for `sdk/typescript`